src/dnshelper.cpp

```diff
@@ -126,13 +126,5 @@ bool DNSHelper::shouldExcludeDNS() {
     return false;
   }
 
-  if (isLocalDNS && settings->localNetworkAccess()) {
-    // DNS is lan, but we already excluded local-ip's, all good.
-    return false;
-  }
-
-  if (!isLocalDNS) {
```

Comment:
This should be: `return false;` Why true?

Reply:
I've updated the function to never exclude local dns (return false for all local dns) because we're either a) on a platform that always excludes them elsewhere or b) on a platform that can't support excluding them (iOS)

Does that look right now?